scheduler/pkg/klcpermit/workflow_manager.go

```diff
@@ -7,15 +7,17 @@ import (
 	"strings"
 
 	"github.com/keptn/lifecycle-toolkit/scheduler/pkg/tracing"
+	"github.com/sirupsen/logrus"
```

Comment:
I would rather stick with the logger implementation provided by `klog` and don't have two implementations